src/crimson/net/ProtocolV2.cc

```diff
@@ -161,10 +161,12 @@ void ProtocolV2::start_connect(const entity_addr_t& _peer_addr,
   conn.target_addr = _peer_addr;
   conn.set_peer_type(_peer_type);
   conn.policy = messenger.get_policy(_peer_type);
-  logger().info("{} ProtocolV2::start_connect(): peer_addr={}, peer_type={},"
+  client_cookie = ceph::util::generate_random_number<uint64_t>(1, -1ll);
```

Comment:
why stop using the helper function of `generate_client_cookie()`?

Reply:
sorry, forget to use the helper function during rebase..